packages/gatsby-theme-docs/src/utils/markdown-fragment-to-react.js

```diff
@@ -70,7 +70,7 @@ const markdownFragmentToReact = (markdownString, customElements) =>
         strong: Markdown.Strong,
         delete: Markdown.Delete,
         hr: Markdown.Hr,
-        img: Image,
+        img: ReleaseNotesImage,
```

Comment:
If the image component must be renamed it should be `MarkdownFragmentToReactImage`.

Reply:
`MarkdownFragmentImage` would suffice there but I agree it's not a generic sort of image but specific to being in Markdown